Makefile

```diff
@@ -130,15 +130,28 @@ swagger:
 
 # Command to run Livepeer Realtime AI Video in a Box
 .PHONY: box
-box: livepeer box-runner
+box: box-rebuild
 	./box/box.sh
 
+.PHONY: box-rebuild
+box-rebuild:
+ifeq ($(strip ${REBUILD}),false)
```

Comment:
WDYT of the default behavior being to build only if the artifacts are not already there? Then one can force always/never with the env instead

Reply:
I'm not fun of this explicit automation. So, I think it's ok-enough to rebuild everything by default and if you don't want it (e.g. I don't want it on my TensorDock), then just disable.